setup_dev_env.sh

```diff
@@ -30,5 +30,6 @@ export INFLUX_HOST=$LH
 export RETHINK_HOST=$LH
 
 export REDIS_HOST=$LH
+export DEFAULT_LIMIT=$DEFAULT_LIMIT
```

Comment:
what's the thinking behind this? where is `$DEFAULT_LIMIT` coming from? I suppose I'm missing something :-)